### ISSUE TITLE
No output video after 200k iterations on the lego dataset.

### ISSUE THREAD

**Issue by Txusheng** [yenchenlin/nerf-pytorch#14]
I have followed the instructions and run the "python run_nerf.py --config configs/lego.txt" command.
After 200k iterations ( the instruction claims 100k iterations), no video is output at the "logs/lego_test/lego_test_spiral_100000_rgb.mp4".

The process ended with:
![image](https://user-images.githubusercontent.com/17637838/92428829-c50af300-f1c2-11ea-82ff-9639bd097a2d.png)
It seems that the process exited correctly.

Where should I do to acquire the rendered video, or the output results in other formats?

**Comment by Txusheng** [comment]
I used the option "--render_only" to render the images and videos, and found that all rendered images and video are blank white. Does it seem that the model learns nothing about the lego scene?

**Comment by Txusheng** [comment]
I just ran the program on the fern dataset. The video is rendered successfully.

**Comment by yenchenlin** [comment]
Hello @Txusheng , I've reverted my code and tested it myself.
May you run `python run_nerf.py --config configs/lego.txt --i_video 1000` and check your logs?

**Comment by Txusheng** [comment]
> Hello @Txusheng , I've reverted my code and tested it myself.
> May you run `python run_nerf.py --config configs/lego.txt --i_video 1000` and check your logs?

Thank you for your replying.

It works correctly! The video rendered at 1000th iteration is vague, but the target object can be easily recognized. I guess after 100K iterations I can get the desired rendered video.

By the way, do you have any idea why the video rendered with "python run_nerf.py --config configs/lego.txt" is blank white?

**Comment by yenchenlin** [comment]
Before I replied to this issue, the master branch hosted the code now in [dev](https://github.com/yenchenlin/nerf-pytorch/commits/dev) branch. I believe some commits there messed up the original code and that's why I revered the code.

Fundamentally, `python run_nerf.py --config configs/lego.txt --i_video 1000` is just the same as `python run_nerf.py --config configs/lego.txt` but with different video rendering frequency.
